src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -1,6 +1,6 @@
 <app-header></app-header>
 <div class="container-fluid screen-min-ht">
-  <div *ngIf="workItem">
+  <div *ngIf="workItem" id="workItemDetail_Wrapper">
   <h1>{{workItem.name}} Details</h1>
   <br />
   <div class="form-horizontal">
```

Comment:
Do we need IDs for labels like ID, Title, Description etc?
We need to add ID for workitem ID 
Line 58 and 59 - Buttons Save and Back need to have IDs